crates/hyperswitch_connectors/src/connectors/placetopay/transformers.rs

```diff
@@ -125,6 +126,7 @@ impl TryFrom<&PlacetopayRouterData<&types::PaymentsAuthorizeRouterData>>
                         .get_card_expiry_month_year_2_digit_with_delimiter("/".to_owned())?,
                     cvv: req_card.card_cvc.clone(),
                 };
+                let return_url = item.router_data.request.complete_authorize_url.clone();
```

Comment:
Is the request body same for 3ds and non-3ds request?
Should we pass complete_authorize_url in case of non-3ds payments?